### PR TITLE
MGMT-3358 Image CRD name convention fixes

### PR DESCRIPTION
**Pull request by nmagnezi**

### CODE REVIEW

File: internal/controller/api/v1alpha1/image_types.go [hunk 1]
```diff
@@ -27,16 +27,17 @@ type ImageSpec struct {
 	ClusterRef   *corev1.ObjectReference `json:"clusterRef"`
 	SSHPublicKey string                  `json:"sshPublicKey,omitempty"`
 	// The name of the secret containing the pull secret
-	PullSecretRef         *corev1.SecretReference `json:"pullSecretRef"`
-	IgnitionOverrides     string                  `json:"ignitionOverrides,omitempty"`
-	StaticIpConfiguration string                  `json:"staticIpConfiguration,omitempty"`
+	PullSecretRef     *corev1.SecretReference `json:"pullSecretRef"`
+	IgnitionOverrides string                  `json:"ignitionOverrides,omitempty"`
+	StaticIpsConfig   string                  `json:"staticIpConfiguration,omitempty"`
```
Comment (filanov) [inline internal/controller/api/v1alpha1/image_types.go:32]: We talked about it in a different PR but it should be a struct right?

Comment (RazRegev) [reply internal/controller/api/v1alpha1/image_types.go:32]: I changed it in my image controller pr to be a struct

**Comment by filanov** [comment]
@nmagnezi is this PR still relevant? i think that https://github.com/openshift/assisted-service/pull/865 is doing the same no?

File: internal/controller/api/v1alpha1/image_types.go (hunk 1 above)
Comment (nmagnezi) [reply internal/controller/api/v1alpha1/image_types.go:32]: > @nmagnezi is this PR still relevant? i think that #865 is doing the same no?

@filanov yup. I missed that. closing this PR.

**Comment by nmagnezi** [comment]
implemented in https://github.com/openshift/assisted-service/pull/865

**Pull request closed without merging**